### PR TITLE
Destination MySQL: bump version

### PR DESCRIPTION
**Pull request by tuliren**
## What
- Bump destination MySQL version to pick up base normalization change.
- This is a follow up PR for #4163.

### CODE REVIEW

File: docs/integrations/destinations/mysql.md [hunk 1]
```diff
@@ -56,3 +56,14 @@ You should now have all the requirements needed to configure MySQL as a destinat
 * **Password**
 * **Database**
 
+## CHANGELOG
+
+| Version | Date | Pull Request | Subject |
```
Comment (davinchia) [inline docs/integrations/destinations/mysql.md:61]: Thanks for this!

**Review by davinchia — APPROVED** [review]

**Comment by tuliren** [comment]
Hmm, publishing failed even with disabled cache:

```
> Failed to apply plugin 'airbyte-python'.
   > A problem occurred configuring project ':airbyte-integrations:connectors:source-zoom-singer'.
      > Could not open cp_proj generic class cache for build file '/home/runner/work/airbyte/airbyte/airbyte-integrations/connectors/source-zoom-singer/build.gradle' (/home/runner/.gradle/caches/6.7.1/scripts/4ffd4lk4g399iqzx3xoc30xgr).
         > java.lang.StackOverflowError (no error message)
```

**Pull request merged**